sklearn/ensemble/_stacking.py

```diff
@@ -967,7 +967,12 @@ def fit(self, X, y, sample_weight=None):
             Returns a fitted instance.
         """
         _raise_for_unsupported_routing(self, "fit", sample_weight=sample_weight)
-        y = column_or_1d(y, warn=True)
+        try:
+            # Single Target
+            y = column_or_1d(y, warn=True)
+        except ValueError:
+            # Multioutput target.
+            y = self._validate_data(X="no_validation", y=y, multi_output=True)
         return super().fit(X, y, sample_weight)
```

Comment:
I think this might break if any of the estimators that we specified does not support multioutput. Maybe we should also check that if multioutput is specified when any of the estimators does not multioutput and raise an error. Moreover we might also need to add this enhancement in the _BaseStacking class's fit method because it currently does not specify that y can have a multioutput shape.